proseco/core.py

```diff
@@ -561,6 +562,34 @@ def from_pickle(cls, obsid, rootdir='.'):
         with open(infile, 'rb') as fh:
             return pickle.load(fh)
 
+    def has_column_spoiler(self, cand, stars, mask=None):
+        """
+        Returns True if ``cand`` has a column spoiler downstream toward readout
+        register which is:
+        - at least 4.5 mag brighter (CHAR.col_spoiler_mag_diff)
+        - within 10 pixels in column (CHAR.col_spoiler_pix_sep)
+
+        Also records a log entry for detected spoilers
+
+        :param cand: object with 'row', 'col', and 'mag' items
+        :param stars: StarsTable
+        :param mask: boolean mask of stars to check (default=None => no mask)
+        :returns: bool
+        """
+        if mask is None:
+            mask = ()  # No mask
+
+        bads = ((np.abs(cand['row']) < abs(stars['row'][mask])) &  # Further from readout register
```

Comment:
Not clear if the comment is talking about the candidate star or the possible spoiler.

Reply:
Possible spoiler.  BTW I coded this differently from you to avoid a possible divide by 0 in your `direction` variable if the row is exactly 0.0.  In practice it is unlikely to ever occur in the remainder of the mission, but still..

Reply:
Good catch.